source/documentation/tools.md

```diff
@@ -233,6 +275,8 @@ If you have tried [Reset your environment to default](#reset-your-environment-to
 
 2. In Control Panel, for R Studio, select the "Restart" button
 
+It can be useful to do this if you have tried [Reset your conda environment to default](#reset-your-conda-environment-to-default) and are still having problems.
```

Comment:
```suggestion
It can be useful to do this if you have tried to [reset your conda environment to default](#reset-your-conda-environment-to-default) and are still having problems.
```